packages/trpc/server/routers/viewer/webhook/testTrigger.handler.ts

```diff
@@ -10,7 +10,7 @@ type TestTriggerOptions = {
 };
 
 export const testTriggerHandler = async ({ ctx: _ctx, input }: TestTriggerOptions) => {
-  const { url, type, payloadTemplate = null } = input;
+  const { url, type, payloadTemplate = null, secret = null } = input;
```

Comment:
Removing the `secret` here because is not defined on the `sendPayload` webhook argument:

```ts
webhook: Pick<Webhook, "subscriberUrl" | "appId" | "payloadTemplate">,
```